src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerChildListValidator.java

```diff
@@ -54,12 +55,15 @@ public boolean isValidLocations(CaseWorkerProfile caseWorkerProfile, ConstraintV
         if (isNotEmpty(caseWorkerProfile.getLocations())
             && caseWorkerProfile.getLocations().size() > 1) {
             //@TO do remove getLocationName with Id problem with excel sheet
-            isValidLocations = negate(caseWorkerProfile.getLocations().get(0).getLocationName()
-                .equalsIgnoreCase(caseWorkerProfile.getLocations().get(1).getLocationName()));
+            isValidLocations = negate(
+                    ObjectUtils.nullSafeEquals(
+                            caseWorkerProfile.getLocations().get(0).getLocationName(),
+                            caseWorkerProfile.getLocations().get(1).getLocationName()));
```

Comment:
Proposed change does not preserve equality while ignoring case as it was before. A possible solution would be to wrap each location name with:
String.valueOf(caseWorkerProfile.getLocations().get(0).getLocationName()).toLowerCase(),
String.valueOf(caseWorkerProfile.getLocations().get(1).getLocationName()).toLowerCase()

Reply:
Good spot, although I'm starting to think a prior null check might be cleaner, I'll run with your solution for now